RadeonRays/CMakeLists.txt

```diff
@@ -232,10 +232,10 @@ if (UNIX AND NOT APPLE)
         string(REGEX MATCH "[0-9]*\.[0-9]*$" RR_API_VERSION ${RR_API_VERSION})
         
         set_target_properties(RadeonRays PROPERTIES SOVERSION ${RR_API_VERSION})
-elseif (APPLE AND CMAKE_OSX_ARCHITECTURES=x86_64)
-    target_compile_options(RadeonRays PUBLIC -stdlib=libc++ -msse4.1 -mmacosx-version-min=10.12)
 elseif (APPLE AND CMAKE_OSX_ARCHITECTURES=arm64)
     target_compile_options(RadeonRays PUBLIC -stdlib=libc++ -mmacosx-version-min=10.12)
```

Comment:
Apple Silicon is only supported on Big Sur and later (10.16 or 11.0), -mmacosx-version-min=10.12 doesn't make sense.

Reply:
Fix the min macOS version to 11.0 for the arm64 build. For the x86_64 build, 10.12 is what Apple suggests to be the min OS version now.